### PR TITLE
[POR-1798] Porter yaml v2 porter app run command

### PR DESCRIPTION
**Pull request by ferozemohideen**

### CODE REVIEW

File: cli/cmd/commands/app.go [hunk 1]
```diff
@@ -43,6 +43,11 @@ var (
 	appMemoryMi      int
 )
 
+const (
+	cnb_life_cycle_launcher = "/cnb/lifecycle/launcher"
```
Comment (ianedwards) [inline cli/cmd/commands/app.go:47]: tiny nit: not sure we ever do snake case in the Go code

File: cli/cmd/commands/app.go [hunk 2]
```diff
@@ -416,17 +441,32 @@ type appPodSimple struct {
...
+	if resp == nil {
+		return nil, false, errors.New("get pods response is nil")
+	}
 	pods := *resp
 
+	if len(pods) == 0 {
+		return nil, false, errors.New("no running pods found for this application")
+	}
+
+	var containerHasLauncherStartCommand bool
```
Comment (ianedwards) [inline cli/cmd/commands/app.go:462]: could move containerHasLauncherStartCommand to the top and return zero value on error

**Review by ianedwards — APPROVED** [review]
nice

File: cli/cmd/commands/app.go (hunk 1 above)
Comment (ferozemohideen) [reply cli/cmd/commands/app.go]: renamed

**Pull request merged**